ssllabsscan/ssllabs_client.py

```diff
@@ -59,7 +59,7 @@ def analyze(self, host, summary_csv_file):
         # write the summary to file
         self.append_summary_csv(summary_csv_file, host, data)
 
-    def start_new_scan(self, host, publish="off", startNew="off", all="done", ignoreMismatch="on"):
+    def start_new_scan(self, host, publish="off", startNew="on", all="done", ignoreMismatch="on"):
```

Comment:
Yeah, would be good to change the default value of `startNew` back to `off`.

Reply:
Yes, I prefer to have that setting as off, so that a new scan will only be started if there's no cache for the host.